Appraisals

```diff
@@ -23,3 +28,9 @@ appraise "activerecord_6_0" do
   gem "activerecord",  "~> 6.0.0"
   gem "activesupport", "~> 6.0.0"
 end
+
+appraise "activerecord_6_1" do
+  gem "activerecord",  "~> 6.1.0"
+  gem "activesupport", "~> 6.1.0"
+  gem "pg", "~> 1.1"
+end
```

Comment:
missing line break here. Could you add trailing line break for consistency, please

Reply:
🙆‍♂️ done